pymc/distributions/multivariate.py

```diff
@@ -405,6 +409,15 @@ def dist(cls, a, **kwargs):
 
         return super().dist([a], **kwargs)
 
+    def get_moment(rv, size, a):
+        norm_constant = at.sum(a, axis=-1)[..., None]
+        moment = a / norm_constant
+        if not rv_size_is_none(size):
+            if isinstance(size, int):
```

Comment:
Actually we may need to revisit this. When size is a number it is still inside a TensorConstant. Not sure what `isinstance(size, int)` will do. Maybe we should check for `size.ndim`?

Reply:
> Looks great Larry! Thanks a lot

As always, thanks to you for all your help.

Should we change `isinstance(size, int)` to `if isinstance(size, int) or size.ndim > 0`? Would that be a correct approach?

Reply:
I checked again and I think that checks will never evaluate true. Even when size is an integer, it gets converted to a `TensorConstant([size])`:

```python
x = at.random.dirichlet([1, 2, 3], size=2)
size = x.owner.inputs[1]
isinstance(size, int)  # False
```

If I remove the `isinstance` block, all tests still pass